### ISSUE TITLE
The VS Code panel gets stuck "loading..." the status of VMs

### ISSUE THREAD

**Issue by salgiza** [microsoft/vscode-azurevirtualmachines#337]
This is something that started happening since the update that removed the Virtual Machines panel and integrated them in a single Resources panel.

I have the panel grouped by resource group.
When I expand the resource group, first time in the morning, the first time that I do it, there's like a 50% chance that the resource will show the "loading..." message next to it (it's updating the status of the VM)... and will never stop loading. Reloading the window and trying again does fix it.

I can't really give any extra steps to reproduce it, because there is nothing particular to what I do: I just go to the Resources panel, expand the subscription, expand the resource group, and that's it.

I have the Azure Account, Azure Resources, Azure Virtual Machines and Azure Functions extensions installed.

I don't know whether this will be enough to investigate it or not, but at least I thought that I should let you know about it, because this bug is quite annoying.

**Comment by alexweininger** [comment]
Thanks for bringing this to our attention. We can see how this would be quite annoying, so we'll look into fixing this ASAP.

Have you observed this behavior with any resources other than Virtual Machines?

Lastly, could you share what version of the Azure Resources and Azure Virtual Machines extensions you have installed?

**Comment by salgiza** [comment]
Hi @alexweininger,

Thanks for your quick response. Sorry that I didn't add anything else to the ticket: after I posted the tickets there were a couple of days that I could not look into it and, when I did... I could not reproduce the bug again (which is quite of ironic, as I had been suffering it *every single day* for quite a while, when I finally decided to report it).

I don't know whether it is because some of the tests/things I did when I reported it did something that fix it for some reason or, most probably, if one of the recent updates to the Azure Resources/Azure Virtual Machines extensions fixed it.

Regardless, I'm going to close the ticket, as there is not much to it now, it seems.

Again, thanks a lot for your support, and for your useful extension.
